### PR TITLE
Add 10.0 delivery carrier label seur

### PR DESCRIPTION
**Pull request by angelmoya**
Check before:

#124

### CODE REVIEW

File: delivery_carrier_label_seur/model/stock.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    def _get_seur_services(self):
+        return self.env['delivery.carrier'].SEUR_SERVICES
+
+    def _get_seur_products(self):
+        return self.env['delivery.carrier'].SEUR_PRODUCTS
+
+    @api.onchange('carrier_id')
+    def carrier_id_change(self):
+        super(StockPicking, self).carrier_id_change()
+        if not self.carrier_id or self.carrier_id.type != 'seur':
```
Comment (jesusVMayor) [inline delivery_carrier_label_seur/model/stock.py:64]: this must be self.carrier_id.carrier_type != 'seur'

**Review by jesusVMayor — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
Hi @angelmoya , could you take a look here please? Are you using this module in customer's production instance? Thanks! :smile:

**Comment by angelmoya** [comment]
Hi @rafaelbn I'm not using on production, this PR has one year, and I did in colaboration with customer that auto implant odoo.... so I don't support this now.

@jesusVMayor please, create new PR with this changes.

**Comment by rafaelbn** [comment]
OK @angelmoya . Please could you at least rebase this PR as https://github.com/OCA/delivery-carrier/pull/124 was merged yo see if bots go green? Thanks!

**Pull request closed without merging**